tests/system/aiplatform/test_vizier.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+
+from google.api_core import exceptions
+from google.cloud import aiplatform
+from tests.system.aiplatform import e2e_base
+from google.cloud.aiplatform.vizier import pyvizier
+
+_TEST_STUDY_ID = 123
+
+
+class TestVizier(e2e_base.TestEndToEnd):
```

Comment:
Decorate with `tear_down_resources` to clean up in case the test fails. example: https://github.com/googleapis/python-aiplatform/blob/main/tests/system/aiplatform/test_experiments.py#L41